crates/biome_markdown_parser/src/syntax.rs

```diff
@@ -15,19 +19,54 @@ pub(crate) fn parse_block_list(p: &mut MarkdownParser) {
     while !p.at(T![EOF]) {
         parse_any_block(p);
     }
-    m.complete(p, MARKDOWN_BLOCK_LIST);
+    m.complete(p, MD_BLOCK_LIST);
 }
 
 pub(crate) fn parse_any_block(p: &mut MarkdownParser) {
```

Comment:
Parsing functions should always return a `ParsedSyntax`. Not returning a `ParsedSyntax` is considered a side effect, which we want to avoid